### PR TITLE
Teachers page

### PR DESCRIPTION
**Pull request by Moradi66**

### CODE REVIEW

**Comment by crdoconnor** [comment]
Are you aware of the conflict?

File: frontend/src/components/form/user/addExperience.jsx [hunk 1]
```diff
@@ -1,141 +1,151 @@
...
+                value={this.state.what_date}
+                onChange={this.handleChange}
+              />
+            </div>
+            <div className="form-group">
+              <Input
+                className="form-control"
+                name="what_place"
+                type="text"
+                placeholder="what_place"
```
Comment (KrustyC) [inline frontend/src/components/form/user/addExperience.jsx]: please change the placeholder to reflect a phrase rather the a field... something like "Insert the place"

File: frontend/src/components/form/user/addExperience.jsx [hunk 2]
```diff
@@ -1,141 +1,151 @@
...
+                value={this.state.what_experience}
+                onChange={this.handleChange}
+              />
+            </div>
+            <div className="form-group">
+              <Input
+                className="form-control"
+                name="what_date"
+                type="text"
+                placeholder="what_date"
```
Comment (KrustyC) [inline frontend/src/components/form/user/addExperience.jsx]: please change the placeholder to reflect a phrase rather the a field... something like "Insert the date"

File: frontend/src/components/teachers/teachersList.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  searchHandler = e => {
+    this.setState({
+      term: e.target.value,
+      result: e.target.value.length ? true : false
+    });
+  };
+
+  render() {
+    const teachers = this.props.teachers;
+    let searchResult = teachers.filter(searchingFor(this.state.term));
```
Comment (KrustyC) [inline frontend/src/components/teachers/teachersList.jsx]: make this a const as you are not changing its value anywhere

File: frontend/src/components/users/editExperience.jsx [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+                value={what_experience}
+                onChange={this.handleChange}
+              />
+            </div>
+            <div className="form-group">
+              <Input
+                className="form-control"
+                name="what_date"
+                type="text"
+                placeholder="what_date"
```
Comment (KrustyC) [inline frontend/src/components/users/editExperience.jsx]: same as abve

File: frontend/src/components/users/editExperience.jsx [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+                value={what_date}
+                onChange={this.handleChange}
+              />
+            </div>
+            <div className="form-group">
+              <Input
+                className="form-control"
+                name="what_place"
+                type="text"
+                placeholder="what_place"
```
Comment (KrustyC) [inline frontend/src/components/users/editExperience.jsx]: same as above

**Pull request merged**